hack/arktos-worker-up.sh

```diff
@@ -32,6 +39,8 @@ KUBELET_KUBECONFIG=${KUBELET_KUBECONFIG:-"${SECRET_FOLDER}/kubelet.kubeconfig"}
 KUBELET_CLIENTCA=${KUBELET_CLIENTCA:-"${SECRET_FOLDER}/client-ca.crt"}
 HOSTNAME_OVERRIDE=${HOSTNAME_OVERRIDE:-"$(hostname)"}
 CLUSTER_DNS=${CLUSTER_DNS:-"10.0.0.10"}
+FEATURE_GATES="${FEATURE_GATES_UP_OUT},MandatoryArktosNetwork=true"
```

Comment:
I am still not clear here: is MandatoryArktosNetwork=true only set for worker? I think it should be true or false for entire cluster.

Reply:
Before this PR, MandatoryArktosNetwork=true has been set in both hack/arktos-up.sh and hack/arktos-up-scale-out-poc.sh when DISABLE_NETWORK_SERVICE_SUPPORT is false.

hack/arktos-up.sh:
line54: FEATURE_GATES="${FEATURE_GATES},MandatoryArktosNetwork=true"

hack/arktos-up-scale-out-poc.sh:
line119: FEATURE_GATES="${FEATURE_GATES},MandatoryArktosNetwork=true"

If DISABLE_NETWORK_SERVICE_SUPPORT is false on the worker node, MandatoryArktosNetwork=true should set in hack/arktos-worker-up.sh.

line42: FEATURE_GATES="${FEATURE_GATES},MandatoryArktosNetwork=true"